glos_qartod/qc.py

```diff
@@ -379,10 +396,18 @@ def get_config(self, variable):
         station_name = self.find_station_name()
         station_id = station_name.split(':')[-1]
         rows = self.config[(self.config['station_id'].astype(str) == station_id) &
+                           (self.config['variable'] == variable) |
+                           (self.config['station_id'].astype(str) == '*') &
                            (self.config['variable'] == variable)]
-        if len(rows) > 0:
-            return rows.iloc[-1]
-        raise KeyError("No configuration found for %s and %s" % (station_id, variable))
+        # prefer station specific variable configuration to generalized variable
+        # configuration, if it is available
+        dedup = rows.sort_values(['variable', 'station_id'], ascending=[True,
+                                         False]).drop_duplicates('variable')
```

Comment:
Could you change this line so that it's more readable?

```python
dedup = rows.sort_values(['variable', 'station_id']],
                         ascending=[True, False])
dedup = dedup.drop_duplicates('variable') 
```